registry.json

```diff
@@ -1649,6 +1649,7 @@
   {
     "id": "heco",
     "name": "ECO Chain",
```

Comment:
Can we change the name?

Reply:
change the name will also change CoinType enum, it's been a while in wallet core